index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ben Omayio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <main>
+        <nav>
+          <a href="#">Kiboma's</a
+          ><a><img src="imgs/showhide_buttons.png" alt="" /></a>
+        </nav>
```

Comment:
### OPTIONAL
_I write it as an optional but this is a requirement that is not mentioned in the first review. Please fix it._


- [ ] This part should be wrapped by a header tag. Please fix it. 

- [ ] Only the navbar icon should be wrapped by the nav tag. You should move the tag outside the nav tag.